document-portal/document-portal-fuse.c

```diff
@@ -3544,6 +3559,7 @@ xdp_fuse_init (GError **error)
   my_gid = getgid ();
 
   all_inodes = g_hash_table_new_full (g_int64_hash, g_int64_equal, NULL, NULL);
+  open_files = NULL;
```

Comment:
The assignment doesn't seem necessary, perhaps a `g_assert (open_files == NULL);` to be extra safe